Sources/NIO/EventLoop.swift

```diff
@@ -50,6 +50,61 @@ public struct Scheduled<T> {
     }
 }
 
+public class RepeatedTask {
+    private let delay: TimeAmount
+    private var scheduled: Scheduled<EventLoopFuture<Void>>? {
+        didSet {
+            reschedule()
+        }
+    }
+    private var task: (() throws -> EventLoopFuture<Void>)?
+
+    internal init(interval: TimeAmount, task: @escaping () throws -> EventLoopFuture<Void>) {
+        self.delay = interval
+        self.task = task
+    }
+
+    /// Try to cancel the execution of the repeated task.
+    ///
+    /// Whether the execution of the task is immediately canceled depends on whether the execution of a task has already begun.
+    ///  This means immediate cancellation is not guaranteed.
+    public func cancel() {
+        scheduled?.cancel()
```

Comment:
please prefix those (and the following) by `self.` (won't repeat this comment for all the other methods :) )